go/private/rules/transition.bzl

```
@@ -54,6 +64,14 @@ def filter_transition_label(label):
     else:
         return str(Label(label))
 
+def _original_value_setting(setting):
+    if not "//go/config:" in setting:
+        return None
+    name = setting.split(":")[1]
+    return filter_transition_label("@io_bazel_rules_go//go/private/rules:original_" + name)
+
+_ORIGINAL_VALUE_SETTINGS = [_original_value_setting(setting) for setting in POTENTIALLY_TRANSITIONED_SETTINGS]
```

```suggestion
_ORIGINAL_SETTING_KEYS = [_original_value_setting(setting) for setting in POTENTIALLY_TRANSITIONED_SETTINGS]
```

Also, because `_original_value_setting` may return `None`. You need to filter them out